### PR TITLE
Stand-alone README and code

### PR DESCRIPTION
**Pull request by tomersolomon1**
Fix the README and make the main branch to be self-contained.

Signed-off-by: tomersolomon1 <tomer.solomon1@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 include Makefile.env
 
-export DOCKER_USERNAME ?= mohammad-nassar10
+export DOCKER_USERNAME ?= tomersolomon1
```
Comment (roytman) [inline Makefile:3]: we don't need this line

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 export DOCKER_PASSWORD ?= 
 export DOCKER_HOSTNAME ?= ghcr.io
-export DOCKER_NAMESPACE ?= mohammad-nassar10
+export DOCKER_NAMESPACE ?= tomersolomon1
```
Comment (roytman) [inline Makefile:6]: it should be fybrik

File: dremio-module.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   type: service
   chart:
-    name: ghcr.io/mohammad-nassar10/dremio-module-chart:0.0.0
+    name: ghcr.io/tomersolomon1/dremio-module-chart:0.0.0
```
Comment (Mohammad-nassar10) [inline dremio-module.yaml:12]: It should be `ghcr.io/fybrik/dremio-module-chart:0.0.0`

File: dremio-module/values.yaml [hunk 4]
```diff
@@ -14,10 +14,41 @@ image:
     pullSecret: null
 
 imageChart: 
-  repository: ghcr.io/mohammad-nassar10/dremio-module
+  repository: ghcr.io/tomersolomon1/dremio-module
```
Comment (Mohammad-nassar10) [inline dremio-module/values.yaml:17]: `ghcr.io/fybrik/dremio-module`

File: README.md [hunk 5]
```diff
@@ -31,17 +31,27 @@ kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/polic
 while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
 ```
 
-### Deploy Fybrik application which triggers the module
+### Deploy Fybrik application
```
Comment (roytman) [inline README.md:34]: in the above section "### Install fybrik" add link.

File: README.md [hunk 6]
```diff
@@ -31,17 +31,27 @@ kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/polic
 while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
```
Comment (roytman) [inline README.md:31]: we should add some explanation about the policies

File: README.md [hunk 7]
```diff
@@ -31,17 +31,27 @@ kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/polic
...
-### Deploy Fybrik application which triggers the module
+### Deploy Fybrik application
 ```bash
 kubectl apply -f fybrikapplication.yaml -n default
 ```
-Run the following command to wait until the fybrikapplication be ready.
+Wait for the fybrik module:
 ```bash
-while [[ $(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
+while [[ ($(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true") || ($(kubectl get jobs my-notebook-default-dremio-module -n fybrik-blueprints -o 'jsonpath={.status.conditions[0].type}') != "Complete") ]]; do echo "waiting for FybrikApplication" && sleep 5; done
```
Comment (roytman) [inline README.md:40]: we can add a comment here, that the operation can take several minutes

Comment (roytman) [reply README.md:40]: Dremio doesn't workk for me

fybrik-blueprints    dremio-master-0                              0/1     Init:3/4   0          3m48s

**Review by roytman — APPROVED** [review]

**Pull request merged**